### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/client-platforms/socket-io/01-login.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
```
Comment (mpaktiti) [inline articles/client-platforms/socket-io/01-login.md:14]: @mehreencs87 Please add the following line:
`githubUrl: 'https://github.com/auth0-samples/auth0-socket.io-samples/tree/master/00-Starter-Seed'`

See the updated [contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects) for details.

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by mehreencs87** [comment]
@mpaktiti githuburl line added.

**Comment by mpaktiti** [comment]
Hi @mehreencs87 
Please rebase to resolve the conflicts. Also, you need to add one more line at the include: `pkgOrg: 'auth0-samples',`
Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

**Comment by mehreencs87** [comment]
@mpaktiti resolved here https://github.com/auth0/docs/pull/2070

**Pull request closed without merging**